airbyte-integrations/singer/local_csv/destination/src/test-integration/java/io/airbyte/integration_tests/destinations/TestLocalCsvDestination.java

```diff
@@ -34,7 +34,6 @@
 import io.airbyte.config.StandardCheckConnectionInput;
 import io.airbyte.config.StandardCheckConnectionOutput;
 import io.airbyte.workers.DefaultCheckConnectionWorker;
-import io.airbyte.workers.DefaultDiscoverCatalogWorker;
```

Comment:
At some point we need to stop using workers in the integration tests. Could be a good follow up task

Reply:
I'll create an issue.